src/test/java/com/conveyal/gtfs/util/PolylineUtilsTest.java

```diff
@@ -109,6 +109,6 @@ public void simplify_returnSameListWhenListSizeIsLessThanOrEqualToTwo(){
         path.add(gf.createPoint(new Coordinate(0, 0)));
         path.add(gf.createPoint(new Coordinate(10, 0)));
         List<Point> simplifiedPath = simplify(path, PRECISION_6, true);
-        assertTrue("Returned list is different from input list", path == simplifiedPath);
+        Assertions.assertTrue(path == simplifiedPath, "Returned list is different from input list");
```

Comment:
I think for any assert method, it's probably OK to just do the static import.

Reply:
yeah, missed a few of these. Have addressed.